packages/liveblocks-client/src/LiveList.ts

```diff
@@ -104,6 +110,21 @@ export class LiveList<TItem extends Lson> extends AbstractCrdt {
     return ops;
   }
 
+  /**
+   * @internal
+   *
+   * Adds a new item into the sorted list, in the correct position.
+   */
+  _insortItem(item: LiveNode) {
```

Comment:
"Insort" is not a typo. I took the name from Python's equivalent stdlib function: [`insort()`](https://docs.python.org/3/library/bisect.html#bisect.insort) (from the `bisect` module). That `bisect` module from Python offers other interesting helper functions for maintaining an always-sorted list internally, which we might use to avoid having to sort the items list every time we change an item. (An optimization for later, though. Not for now.)

Reply:
Okay, that name may actually make more sense. Updated in c63ad5fcd7953b2bf20c135d0a1f7ade887aae4e! 🙏